Makefile

```diff
@@ -5,6 +5,7 @@ export CURRENT_UID
 export NODE_ENV ?= development
 
 DC_DEV := docker-compose -p usine-distribuee
+DOCKER_ADMIN := docker run --rm -v ${PWD}/apps/admin:/admin -u=${CURRENT_UID} -w /admin node:13.11.0
```

Comment:
How do we handle env vars with this?
For dev, I set up `API_URL` in docker-compose, but here?

Reply:
@floo51 yes, we'd have to inject it into the container. Or we could do a `host.js` : 

```
export const apiUrl = process.env.NODE_ENV === 'production' ? '/api' : process.env.API_URL;
```
(as in production, it'll be the same domain !)
What do you refer ?